### PR TITLE
update Viaplay.Tv

### PR DESCRIPTION
**Pull request by davidclaeysquinones**
This commit removes SBS9 as it no longer exists and replaces it by Viaplay Tv

### CODE REVIEW

**Comment by BellezaEmporium** [comment]
If SBS9.nl doesn't exist anymore, better keep the channel in, mark it as closed, and add the other channel.

**Comment by davidclaeysquinones** [comment]
I've added SBS9 so that it is preserved.

File: data/channels.csv [hunk 1]
```diff
@@ -24910,6 +24910,7 @@ SBNTVInternational.us,SBN TV International,,,,US,,,r/INT,eng;spa;som,religious,F
 SBS.us,SBS,Shinniigaa Broadcasting Service,,,US,,,r/AFR,som,news,FALSE,,,,,https://i.imgur.com/bOHMrky.png
 SBS6.nl,SBS 6,,,,NL,,,c/NL,nld,,FALSE,,,,https://www.sbs6.nl/,https://i.imgur.com/pCfKkqB.png
 SBS9.nl,SBS 9,,,,NL,,,c/NL,nld,,FALSE,,,,https://www.sbs9.nl/,https://i.imgur.com/8AeOOvm.png
+ViaplayTv.nl,Viaplay TV,,,,NL,,,c/NL,nld,,FALSE,,,,https://nl.wikipedia.org/wiki/Viaplay_TV#/media/Bestand:Viaplay_TV_logo.svg/2
```
Comment (BellezaEmporium) [inline data/channels.csv:24913]:
```suggestion
ViaplayTV.nl,Viaplay TV,,,,NL,,,c/NL,nld,,FALSE,,,,https://nl.wikipedia.org/wiki/Viaplay_TV#/media/Bestand:Viaplay_TV_logo.svg/2

```

**Comment by Dum4G** [comment]
It's better to process this change via editing capabilities of iptvbot

**Pull request closed without merging**